docs/release-notes/NuGet-5.9.md

```diff
@@ -0,0 +1,250 @@
+---
+title: NuGet 5.9 Release Notes
+description: Release notes for NuGet 5.9 including new features, bug fixes, and DCRs.
+author: erdembayar
+ms.author: eryondon
+ms.date: 3/11/2021
+ms.topic: conceptual
+---
+
+# NuGet 5.9 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.9**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.9](https://visualstudio.microsoft.com/downloads/) | [5.0](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
+  
+> [!NOTE]
+> Visual Studio 16.9, MSBuild 16.9, and .NET 5.0 require NuGet.exe 5.9 or later.
+
+## Summary: What's New in 5.9
+
+* Show requested version in the Solution Level UI - [#9827](https://github.com/NuGet/Home/issues/9827)
+
+* Extend the `.nupkg.metadata` file to include the installation source - [#10354](https://github.com/NuGet/Home/issues/10354)
+
+* Add Update context menu item for package dependencies that launches Package Manager UI with preselected packages to update - [#10378](https://github.com/NuGet/Home/issues/10378)
+
+* Enable recommender for PackageReference style projects - [#10053](https://github.com/NuGet/Home/issues/10053)
+
+* Introduce a new msbuild property to exclude build output for specific tfms during pack task - [#10396](https://github.com/NuGet/Home/issues/10396)
+
+* INuGetPackageFileService - Fetch Images and embedded licenses for Codespaces-connected and standalone - [#10151](https://github.com/NuGet/Home/issues/10151)
+
+### Issues fixed in this release
```

Comment:
Just a future thought for everyone.

Although we use GitHub labels to generate these headers like "DCRs, Nones, StillOpens", they have little to no meaning to people reading our notes. 

Keeping it simple as "Features" and "Bugs" I think would help this in the future.

Reply:
I agree, the nones and still opens should not be a thing tbh (those 2 labels don't even exist :D )

Features matters, DCRs matter sometimes because they require user awareness. 
The rest should just be treated equivalently.

Reply:
If we put DCR, we need to spell it out. "Design Change Request" -> "Feature" imo 😎

Reply:
yeah, I agree. 
DCR makes sense for our labels, but not something super release notes friendly.

Reply:
Added in parenthesis this time around.